docs/pages/x/api/charts/gauge-container.json

```diff
@@ -23,6 +23,7 @@
       "type": { "name": "union", "description": "number<br>&#124;&nbsp;string" },
       "default": "'100%'"
     },
+    "skipAnimation": { "type": { "name": "bool" } },
```

Comment:
A consequence of adding the animation plugin as a core plugin is that the gauge chart, which isn't animated, will accept a `skipAnimation` prop. 

Should I remove the animation plugin from the core plugins and apply it separately to all charts? Or this is a good time to add some animation to the gauge chart? 😛

Reply:
I'll take a look 👍